packages/react-slider/src/utils/calculateSteps.ts

```diff
@@ -14,7 +14,7 @@ export const calculateSteps = (
   dir: 'ltr' | 'rtl',
 ): number => {
   const currentBounds = railRef?.current?.getBoundingClientRect();
-  const sliderSize = vertical ? currentBounds!.height : currentBounds!.width;
+  const sliderSize = vertical ? currentBounds!.height || 0 : currentBounds!.width || 0;
```

Comment:
You could also potentially do:

```suggestion
  const sliderSize = (vertical ? currentBounds!.height : currentBounds!.width) || 0;
```